### ISSUE TITLE
Solar Panel energy production is much worse then real-world production

### ISSUE THREAD

**Issue by pwalsh0** [CleverRaven/Cataclysm-DDA#68526]
### Describe the bug

The normal solar panels in-game produce 50W: https://cdda-guide.nornagon.net/vehicle_part/solar_panel

This is absolutely pathetic compared to a normal 1m2 consumer panel that many of us use today. It's quite common these days for people to have 5kW solar setups, or even larger. To get 5kw in-game would require 100 of these panels, and covering the entire roof (after looting an entire city worth of panels).  

A more realistic value for this size of panel should be a value from 250W-500W. 

Advanced panels should be adjusted in similar fashion. 

### Attach save file

N/A

### Steps to reproduce

N/A

### Expected behavior

Solar panels in game should produce similar output to average real photovoltaic panel of similar size.

### Versions and configuration

N/A

**Comment by pwalsh0** [comment]
A realistic figure for 1m2 of avg PV panel today is 150W actually. I assume the ones in-game are bit larger than 1m2.

**Comment by RenechCDDA** [comment]
Some related conversations here: #67508

**Comment by oosyrag** [comment]
Tldr Solar panel tiles are currently sized for vehicles, which means they are not one square meter worth of panel.

**Comment by Standing-Storm** [comment]
> Tldr Solar panel tiles are currently sized for vehicles, which means they are not one square meter worth of panel per tile

One possible solution for this might be editing the Construction solar panel recipes so that the resulting appliance provide a higher power level but takes more than one "panel" to place while leaving vehicle panels with the same 50w/1 panel requirement. That way building solar panels take up a realistic amount of space on the roof.

(You'd have to get more solar panels from deconstructing solar panels found on other roofs for this all to work out as well)

**Comment by pwalsh0** [comment]
I like the idea of making the "roof" panels require 3+ of these vehicle panels to construct.
